remove unused package dependencies

vishalvvr opened the pull request:
- Numpy removed in [PR#94](https://github.com/redhat-performance/opl/pull/94)
- We are installing PyYaml so pyyaml is not doing anything in specific
- Cannot find any code using paho-mqtt package

Commit:
remove unused dependencies

jhutar commented:
Nice! Could you please remove `numpy` from `core/setup.py` as well?

And maybe also remove `python3-numpy` from `Containerfile`?

Commit:
Fix PR#97 review comment

jhutar commented:
Please fix linter errors (see https://github.com/redhat-performance/opl/actions/runs/6795933190/job/18474995543?pr=97) and I think we are good to go.

Commit:
fix linting issues

Project is now PEP8 compliant after running 'black' python code
formatter.
https://pypi.org/project/black

jhutar committed:
fix: DOCKERFILE_HADOLINT: DL4000 error: MAINTAINER is deprecated

jhutar committed:
fix: jscpd: Clone found

jhutar committed:
fix: jscpd: Clone found

jhutar committed:
fix: jscpd: Clone found

jhutar committed:
fix: pylint: E0102: method already defined line 77 (function-redefined)

jhutar committed:
fix: flake8: F841 local variable 'status_data' is assigned to but never used

jhutar commented:
Thank you Vishal!
